src/api/services/format-date-service.ts

```diff
@@ -0,0 +1,21 @@
+import {Injectable} from '@angular/core';
+import moment from 'moment';
+
+@Injectable()
+export class FormatDate {
+
+    constructor() {}
+
+    setLang(lang:string, messages:Object):void {
+        moment.locale(lang, messages || {});
+    }
+
+    getLang() {
+        return moment.locale();
+    }
+
+    getRelative(time):string {
+        return moment(parseInt(time, 10)).fromNow();
+    }
+
+}
```

Comment:
I create this service so we can format all the dates as we need it... but know that I see it maybe is better to have a pipe, thoughts?